CHANGELOG.md

```diff
@@ -4,7 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 ## [1.1.4.1](https://github.com/unzerdev/php-sdk/compare/1.1.4.0..1.1.4.1)
 ### Added
-*   Added Apple Pay example.
+* Added Apple Pay example.
+
+### Change
```

Comment:
```suggestion
### Changed
```

Reply:
Every category e.g. "Changed", "Fixed", "Added", etc. can only be there once per release.
This is here twice.

Reply:
fixed